.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+# This config was automatically generated from your source code
+# Stacks detected: deps:node:.,test:jest:
+version: 2.1
+orbs:
+  node: circleci/node@5
+jobs:
+  test-node:
+    # Install node dependencies and run tests
+    executor: node/default
+    environment:
+      JEST_JUNIT_OUTPUT_DIR: ./test-results/
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: npm install jest-junit
+      - run:
+          name: Run tests
+          command: npm run test:unit
+      - store_test_results:
+          path: ./test-results/
+  build-node:
+    # Build node project
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: npx prisma migrate dev --schema=./src/prisma/schema.prisma && npm run build
+          environment:
+            DATABASE_URL: postgres://samhoho998:0JzQy5AhTVsi@ep-billowing-breeze-86658300.ap-southeast-1.aws.neon.tech/neondb
```

Comment:
We will revise this again in the future. Postgres credential shouldn't be exposed directly in circle ci yaml config file.